src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.0.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>1.1.0</RoslynSemanticVersion>
```

Comment:
Strictly speaking, we don't *have* do, but since someone else could theoretically take a dependency on the fact that the "core" InteractiveWindowCommands had no attributes in v1.0.0.0, I chose to increment the version.